config/initializers/search_paginate_renderer.rb

```diff
@@ -0,0 +1,44 @@
+class SearchPaginateRenderer < WillPaginate::ActionView::LinkRenderer
+
+  def container_attributes
+    super.except(*[:link_options])
+  end
+
+  protected
+
+  def page_number(page)
+    link_options = @options[:link_options] || {}
+    if page == current_page
+      tag :li, tag(:span, page, class: 'page-link'), class: 'page-item active'
+    else
+      link_options.merge! class: 'page-link', rel: rel_value(page)
+      tag :li, link(page, page, link_options), class: 'page-item'
+    end
+  end
+
+  def previous_or_next_page(page, text, classname)
+    link_options = @options[:link_options] || {}
+
+    if page
+      link_wrapper = link(text, page, link_options.merge(class: 'page-link'))
+      tag :li, link_wrapper, class: 'page-item'
+    else
+      span_wrapper = tag(:span, text, class: 'page-link')
+      tag :li, span_wrapper, class: 'page-item disabled'
+    end
+  end
+
+  def gap
+    tag :li, tag(:i, ELLIPSIS, class: 'page-link'), class: 'page-item disabled'
```

Comment:
@Amansingla97 It says this: `ELLIPSIS` is not defined and hence giving an error in production